app/views/admin/shared/_nested_sidebar.html.erb

```diff
@@ -0,0 +1,29 @@
+<% menu_items.each do |group_name, group| %>
+  <% if group[:children].length == 1 %>
+    <a class="crayons-link crayons-link--block <%= "crayons-link--current" if (deduced_controller(request) == group[:children][0][:controller]) %>"
```

Comment:
Looking forward to Rails 6.1, where we have `class_names` to simplify this a bit. It will look like this:

```ruby
class_list(
  'crayons-link', 
  'crayons-link--block', 
  { 'crayons-link--current': deduced_controller(request) == group[:children][0][:controller] }
)
```